java/surface-placement/README.md

```diff
@@ -1,29 +1,52 @@
-# Surface Placement
-Position graphics above a surface using different Surface Placements.
+# Surface placement
 
-![Elevation Mode App](surface-placement.png)
+Position graphics relative to a surface using different surface placement modes.
+
+![Surface placement app](surface-placement.png)
+
+## Use case
+
+Depending on the use case, data might be displayed at a consistent, absolute height (e.g. flight data recorded relative to sea level), at a relative height to the terrain (e.g. transmission lines positioned relative to the ground), or draped directly onto the terrain (e.g. location markers, area boundaries).
 
 ## How to use the sample
-Simply run the app.
+
+The application loads a scene showing three points that use the individual surface placement rules (Absolute, Relative, and either Draped Billboarded or Draped Flat). Use the toggle to change the draped mode, then explore the scene by zooming in/out and by panning around to observe the effects of the surface placement rules.
 
 ## How it works
-1. Create a GraphicsOverlay.
-2. Set the surface placement mode `GraphicsOverlay.getSceneProperties().setSurfacePlacement(...)`
-	- `DRAPED`, Z value of graphic has no affect and graphic is attached to surface
-	- `RELATIVE`, position graphic using it's Z value plus the elevation of the surface
-	- `ABSOLUTE`, position graphic using only it's Z value
-3. Add graphics to the graphics overlay, GraphicsOverlay.getGraphics.add(Graphic).
-4. Add the graphics overlay to the SceneView, SceneView.getGraphicsOverlays().add(GraphicsOverlay).
+
+1. Create a `GraphicsOverlay` for each placement mode, setting `SceneProperties.setSurfacePlacement`:
```

Comment:
should this be `SceneProperties` or `LayerSceneProperties`?

Reply:
Good catch, thanks!